scripts/build_goreleaser_snapshot.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Allows manual building of packages via goreleaser.
+# Allows building of dirty git state and will not push results to github.
+
+# make script executable independent of path
+SCRIPTDIR=$(dirname "$0")
+pushd "${SCRIPTDIR}" >/dev/null
+pushd ".." >/dev/null
+
+
+GORELEASER_IMAGE=iotmod/goreleaser-cgo-cross-compiler:1.13.5
+REPO_PATH="/build"
+
+docker pull "${GORELEASER_IMAGE}"
+docker run --rm --privileged -v "${PWD}":"${REPO_PATH}" -w "${REPO_PATH}" "${GORELEASER_IMAGE}" goreleaser --rm-dist --snapshot
```

Comment:
Please add a `--skip-publish` (e.g. to not publish a docker image by mistake, if you are logged in into the docker hub)
```suggestion
docker run --rm --privileged -v "${PWD}":"${REPO_PATH}" -w "${REPO_PATH}" "${GORELEASER_IMAGE}" goreleaser --rm-dist --snapshot --skip-publish
```